CHANGELOG.md

```diff
@@ -31,6 +32,7 @@ With this release, `rkt` RPM/dpkg packages should have the following updates:
 
 - Pass `--enable-tpm=no` to configure script, if `rkt` should not use TPM.
 - Use the `--with-default-stage1-images-directory` configure flag, if the default is not acceptable and install the built stage1 images there.
+- Distributions using systemd: install the new file `dist/init/systemd/tmpfiles.d/rkt.conf` in `/usr/lib/tmpfiles.d/rkt.conf`.
```

Comment:
Fedora rkt RPM currently exec `rkt install`: http://pkgs.fedoraproject.org/cgit/rpms/rkt.git/tree/rkt.spec#n273

Should it still do that? If not, how does the tmpfile replace it without rebooting? Is there a way to signal systemd to pick up the new tmpfiles config?